pom.xml

```diff
@@ -50,6 +51,16 @@
          <scope>provided</scope>
       </dependency>
       <dependency>
+         <groupId>org.jboss.arquillian.junit</groupId>
```

Comment:
Used for setting `Arquillian.class` but if I set the literal thing then I can remove it.